src/ekore/harmonics/w1.py

```diff
@@ -3,6 +3,7 @@
 import numba as nb
 import numpy as np
 
+from . import cache as c
```

Comment:
did you ever run this code? I'm afraid (in contrast to what I said earlier) we can not do this, because this look pretty much like a cyclic import

Reply:
Ah, you are right. I missed this too. So here we will have to let the computation be repeated. I will revert the w file changes.

Reply:
No, if we repeat the computation we loose the purpose of this PR - the solution is to compute all auxillary harmonic sums in `get` _first_ and them pass them down to, e.g. `Sm1`

Reply:
In the latest commit, I have removed the cyclic import while ensuring that the half harmonic sums are only used from the cache and not recomputed.